src/server/dfly_main.cc

```diff
@@ -609,6 +610,16 @@ void sigill_hdlr(int signo) {
   exit(1);
 }
 
+void PrintBasicUsageInfo() {
+  const string flags_log_dir = FLAGS_log_dir.CurrentValue();
```

Comment:
We use `absl::GetFlag(FLAGS_log_dir)` to get values in our code base. I actually did not know flags had this method :)